app/packages/state/src/recoil/sidebar.ts

```diff
@@ -116,11 +117,12 @@ export const useTagText = (modal: boolean) => {
 export const useEntries = (
   modal: boolean
 ): [SidebarEntry[], (entries: SidebarEntry[]) => void] => {
+  const ds = useRecoilValue(dataset);
   const [entries, setEntries] = useRecoilStateLoadable(
-    sidebarEntries({ modal, loading: false, filtered: true })
+    sidebarEntries({ modal, loading: false, filtered: true, dataset: ds })
   );
   const loadingEntries = useRecoilValue(
-    sidebarEntries({ modal, loading: true, filtered: true })
+    sidebarEntries({ modal, loading: true, filtered: true, dataset: ds })
```

Comment:
`dataset` look like it is unused. Could also be pulled with `get` in the `sidebarEntries` get function